landscaper/pkg/gardenlet/example/landscaper-resources/installation/00-installation.yaml

```
@@ -0,0 +1,51 @@
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Installation
+metadata:
+  name: gardenlet
+  namespace: ls-system
+spec:
+  blueprint:
+    ref:
+      # reference the component descriptor from the registry stored as an OCI artifact
+      # the component descriptor has internally a reference to the Blueprint
+      # Resolved reference: eu.gcr.io/gardener-project/landscaper/dev/components/component-descriptors/github.com/gardener/gardener:latest
+      repositoryContext:
+        type: ociRegistry
+        baseUrl: eu.gcr.io/gardener-project/landscaper/dev/components
+      componentName: github.com/gardener/gardener # name of the component descriptor
+      version: 'latest'
+      # name of the blueprint within the component descriptor
+      resourceName: blueprint
```

think we should rename the gardenlet blueprint as we will have more blueprints in the future
```suggestion
      resourceName: gardenlet-blueprint
```

---

okay makes sense, same as in the integration tests